packer/ubuntu-14.10-amd64.json

```diff
@@ -185,13 +185,16 @@
         "scripts/common/vmtools.sh",
         "scripts/common/chef.sh",
         "scripts/ubuntu/cleanup.sh",
-        "scripts/common/minimize.sh"
+        "scripts/common/minimize.sh",
+        "scripts/ubuntu/resolv.sh"
```

Comment:
Please don't edit the non ganeti/openstack templates. You need to update the ones that have ``ganeti`` and ``openstack`` in the filename.